products/BellHybrid/services/db/databases/migration/settings_bell/current/34a5f438_Move_gs_onboarding_done_to_global_scope/.meta

```diff
@@ -1,6 +0,0 @@
-{
- "id": "34a5f438-a1fe-4818-af74-d084dc67ddf2",
- "date": "2023-03-02 12:44:53",
- "message": "Move gs_onboarding_done to global scope",
- "parent": 0
-}
```

Comment:
There should be no `down.sql` and `up.sql` files too. As a matter of fact, the whole 'current' folder should be removed by the migration script. I've re-checked it locally and it works the way I described. Are you sure you didn't modify it by a hand?

Reply:
I didn't modify anything. I only executed `python3 tools/db_migration.py commit` from main directory after building BellHybrid and setting the envvar to the build directory (AFAIR as relative dir).

Reply:
I did it for more or less master branch so if there was some file that shouldn't be there it was there in the first place. AFAIU it was added in 950cf78aa9fe3f30d25d5048f0ec46bf5fe2eac0

Reply:
AFAIU now it's ok.